maven-java/kalix-spring-boot-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -42,6 +42,30 @@
       </extension>
     </extensions>
 
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <transformers combine.children="append">
```

Comment:
This will allow the child pom to append transformers to the plugin config defined in `spring-boot-parent`.